core/server/data/schema/schema.js

```diff
@@ -187,7 +187,8 @@ module.exports = {
                     'array',
                     'string',
                     'number',
-                    'boolean'
+                    'boolean',
+                    'object'
```

Comment:
@naz this was needed to be able to add the labs setting using `models.Settings.add`. Noted the type wasn't added back in here when the labs setting was re-added with the `object` type - was that intentional?

Reply:
Not intentional. Good spot :+1: This makes me wonder if there's some test missing or some other way to check if the types listed here are in use anywhere :thinking: